src/db/course.ts

```diff
@@ -81,7 +81,11 @@ async function getAllContent() {
   if (value) {
     return value;
   }
-  const allContent = await db.content.findMany({});
+  const allContent = await db.content.findMany({
+    include: {
```

Comment:
This is prolly incorrect. videoProgress is user specific

Reply:
Yeah, will fix this as well